net/instaweb/rewriter/image_rewrite_filter.cc

```diff
@@ -2136,13 +2136,18 @@ void ImageRewriteFilter::StartElementImpl(HtmlElement* element) {
     }
 
     BeginRewriteImageUrl(element, attributes[i].url);
-  }
-
-  if (element->keyword() == HtmlName::kImg) {
+   
+  
     HtmlElement::Attribute* srcset = element->FindAttribute(HtmlName::kSrcset);
+	  HtmlElement::Attribute* datasrcset = element->FindAttribute(HtmlName::kDataSrcset);
     if (srcset != nullptr) {
       BeginRewriteSrcSet(element, srcset);
     }
+	  if (datasrcset != nullptr) {
```

Comment:
@Lofesa tiny nit on formatting here. I'd love to help you with an easier flow to edit these files; for example vscode makes it easier to use git, and also is able to automatically fix formatting issues for you. Please ping me on Slack if you want me to help you get set with that.

Reply:
Changed